src/containers/DexsContainer/index.tsx

```diff
@@ -214,7 +214,7 @@ export default function DexsContainer({
 		return dexs.map(dex => {
 			return {
 				...dex,
-				volumetvl: !loading ? data.protocols.find(p=>p.name===dex.name).tvlPrevDay: undefined,
+				volumetvl: !loading ? dex.totalVolume24h/data.protocols.find(p=>p.name===dex.name).tvlPrevDay: undefined,
```

Comment:
this is extremely inefficient, its O(n*m) iterating over all protocols each time

Reply:
its way better to instead build an object of {protocolName: tvlPrevDay} and then use that

also better to move this to getStaticProps() since that way you can perform computation server-side and send all data already ready